src/app/containers/Blocks/index.jsx

```diff
@@ -20,6 +20,7 @@ const Blocks = ({ blocks, componentsToRender }) =>
 
     return (
       <Block
+        index={index}
```

Comment:
There are cases, and im sure there will be more in future, where we call Blocks recursively, meaning there could 'theoretically' be infinate blocks with an index < 3. Is this a concern?

E.g. If we call Blocks in caption, to render any optimo blocks, it could contain an image, meaning nomatter where on the page, Images inside captions could have lazy load disabled.

I may be missing something, but thats my understanding anyway

Reply:
Yep, this is true. Initially I thought we needn't worry about this, however this solutions would by no means be future-proof, when we start supporting lists for instance.

Will consider other options...